Cargo.toml

```diff
@@ -123,6 +124,18 @@ chacha20poly1305 = { version = "0.7.1", optional = true }
   optional = true
   features = [ "derive" ]
 
+  [dependencies.p256-crate]
```

Comment:
Didn't notice when dependencies switch to this format. Hope, no dependencies were lost